lib/node_runner_test.go

```diff
@@ -17,10 +19,18 @@ import (
 var (
 	kp      *keypair.Full
 	account *block.BlockAccount
+	tlsKey  *sebaknetwork.KeyGenerator
 )
 
 func init() {
 	kp, _ = keypair.Random()
+
+	dir, err := ioutil.TempDir("/tmp/", "sebak-test")
+	if err != nil {
```

Comment:
`require.Nil` ;)

Reply:
@Geod24 This block is not in part of testing. :)